src/Modules/UngradedModules/FreeModElem.jl

```diff
@@ -22,6 +22,13 @@ function FreeModElem(c::Vector{T}, parent::FreeMod{T}) where T
   return FreeModElem{T}(sparse_coords,parent)
 end
 
+function FreeModElem(i::Int, x::T, parent::FreeMod{T}) where T
+  @assert 1 <= i <= rank(parent)
+  sparse_coords = sparse_row(base_ring(parent), [i], [x]; sort=false)
```

Comment:
```suggestion
  sparse_coords = sparse_row(base_ring(parent), [i], [x])
```
no longer needed due to https://github.com/thofma/Hecke.jl/pull/1502